apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -281,8 +282,17 @@ export const OrganizationWrapper: React.FC<WrapperProps> = ({
   showSecondaryMenu = true,
   showExternalLinks = false,
 }) => {
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = useState(false)
   const router = useRouter()
 
+  useEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
```

Comment:
This could be written in fewer lines but how it is right now can be argued to be more clarifying (your call on what you want to do here)

`useEffect(() => {
  setIsMobile(width < theme.breakpoints.md)
}, [width])`

Reply:
After looking at it again I think we don't need the useEffect/useState for this. The `width` value from `useWindowSize` is causing a render on window width change so we can just have a simple `const isMobile = width < theme.breakpoints.md`.